renovate.json

```diff
@@ -21,6 +21,18 @@
       "matchManagers": ["nuget"],
       "groupName": "NuGet dependencies"
     },
+    {
+      "matchManagers": ["nuget"],
```

Comment:
Not sure how we were able to merge back the branch that updated the xunit package if it meant tests weren't able to run and we have a CI pipeline. I feel we should keep having update PR from renovate and figure out why our pipeline didn't fail, but it failed while running on your IDE.